### ISSUE TITLE
FP useless_attribute:  #[allow(unused)] on import

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang/rust-clippy#5229]
````rust
#[allow(unused)]
use std::path::PathBuf;
fn main() {
    println!("Hello, world!");
}
````
clippy claims that the attribute is unused
````
error: useless lint attribute
 --> src/main.rs:1:1
  |
1 | #[allow(unused)]
  | ^^^^^^^^^^^^^^^^ help: if you just forgot a `!`, use: `#![allow(unused)]`
  |
  = note: `#[deny(clippy::useless_attribute)]` on by default
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#useless_attribute
````

however this is a false positive because it is used to silence the `unused_imports` warning as can be seen here: (only the second import is warned about:)

````rust
#[allow(unused)]
use std::path::Path;

use std::path::PathBuf;

fn main() {
    println!("Hello, world!");
}
````

````
warning: unused import: `std::path::PathBuf`
 --> src/main.rs:4:5
  |
4 | use std::path::PathBuf;
  |     ^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

    Finished dev [unoptimized + debuginfo] target(s) in 0.16s
````

**Comment by flip1995** [comment]
FWIW, we already whitelist `allow(unused_imports)` beside other on this lint, we can just extend this list to also whitelist the `unused` group
